.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "shared/BEST-Energy"]
 	path = shared/BEST-Energy
 	url = git@github.com:olisystems/BEST-Energy.git
+	branch = cl/oli-pycli
```

Comment:
This is a simple hotfix branch, where I made some stuff public.

Reply:
Is it recommended to update main with this fixing branch or rather keep this branch updated with ongoing changes? Both client-dependency-fixing-branch and the current worker main branch should be able to work together despite changes regarding the fixes right?